### PR TITLE
#561 added web interface notifiers handling and changed notification Service suffix

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: config/__init__.py [hunk 1]
```diff
@@ -168,7 +169,13 @@
 
 # notifier
 CONFIG_NOTIFIER = "notifier"
+NOTIFIER_REQUIRED_KEY = "required"
+NOTIFIER_PROPERTIES_KEY = "properties"
+NOTIFIER_IGNORED_REQUIRED_CONFIG = "message"
 CONFIG_NOTIFIER_IGNORE = [CONFIG_TWITTER, CONFIG_WEB, CONFIG_TELEGRAM, CONFIG_REDDIT]
+NOTIFIER_REQUIRED_CONFIG = {
+    "gmail": ["username", "password"]
```
Comment (Herklos) [inline config/__init__.py]: Clean :+1:

File: services/service_creator.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def get_name(cls):
                     await service_instance.prepare()
 
                     # notifier
-                    if hasattr(service_class, "PUSHOVER"):
+                    if isinstance(service_instance, NotifierService):
```
Comment (Herklos) [inline services/service_creator.py]: :+1:

File: tools/dict_util.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+
+def find_nested_value(dict_, field):
```
Comment (Herklos) [inline tools/dict_util.py]: There is no python function to do this ?

Comment (GuillaumeDSM) [reply tools/dict_util.py]: I couldn't find any unfortunatelly

Comment (GuillaumeDSM) [reply tools/dict_util.py]: I only found github libraries, which is not worth getting just for this

Comment (Herklos) [reply tools/dict_util.py]: Yes

Comment (Herklos) [reply tools/dict_util.py]: So that's ok for me

**Pull request merged**